start-database.sh

```
@@ -0,0 +1,83 @@
+  read -p "Should we generate a random password for you? [y/N]: " -r REPLY
+  if ! [[ $REPLY =~ ^[Yy]$ ]]; then
+    echo "Please change the default password in the .env file and try again"
+    exit 1
+  fi
+  # Generate a random URL-safe password
+  DB_PASSWORD=$(openssl rand -base64 12 | tr '+/' '-_')
+  sed -i '' "s#:password@#:$DB_PASSWORD@#" .env
+fi
+
+$DOCKER_CMD run -d \
+  --name $DB_CONTAINER_NAME \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)